docs/machine-learning/how-to-guides/use-gams-for-model-explainability.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Use Generalized Additive Models and shape functions for model explainability in ML.NET
+description: Use Generalized Additive Models and shape functions for model explainability in ML.NET
+ms.date: 12/04/2018
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use Generalized Additive Models in ML.NET to be able to analyze the shape function of my models so that I can understand how my machine learning models make decisions.
+---
+
+When creating machine learning models, it is often not enough to simply make predictions. Often, machine learning developers, decision makers, and those affected by the models need to understand how machine learning models make decisions and which features contribute to their performance. **Generalized Additive Models (GAMs)** are used internally at Microsoft for model explainability to help machine learning developers create high-capacity models that can be easily interpreted by others.
+
+GAMs are a class of **interpretable models** that are linear models where the terms are nonlinear functions, called "shape functions" of a single variable. As linear models, they are easily interpreted but because the models learn functions of features instead of a single weight, they can model more complex relationships than a simple linear model. The resulting GAM predictor has an intercept term that represents the average prediction over the training set, and shape functions that represent the deviation from the average prediction. The shape functions can be inspected by eye to see the response of the model to different values of a feature, and visualized like the graph below. The GAM trainer in ML.NET is implemented using shallow gradient boosted trees (e.g. tree stumps) to learn nonparametric shape functions, and is based on the method described in [Intelligible Models for Classification and Regression.](http://www.cs.cornell.edu/~yinlou/papers/lou-kdd12.pdf) by Lou, Caruana, and Gehrke.
```

Comment:
graph below -> following graph

e.g. -> for example

remove period from the link text
 
use https link